sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -419,9 +437,11 @@ def _add_span_request_attributes(self, span):
     def _get_eventhub_properties(self):
         # type:() -> Dict[str, Any]
         mgmt_msg = Message(application_properties={"name": self.eventhub_name})
-        response = self._management_request(mgmt_msg, op_type=MGMT_OPERATION)
+        # TODO: remove mocked response
+        #response = self._management_request(mgmt_msg, op_type=MGMT_OPERATION)
         output = {}
-        eh_info = response.get_data()  # type: Dict[bytes, Any]
+        eh_info = {b"name": b"eventhub-test", b"created_at": 1608181432000, b"partition_ids": [b'0']}
```

Comment:
we could set it to the array as following to mock receiving from multiple partions:
```python
b"partition_ids": [b'0', b'1', b'2', b'3', b'4', b'5', b'6', b'7', b'8', b'9', b'10', b'11', b'12', b'13', b'14', b'15', b'16', b'17', b'18', b'19', b'20', b'21', b'22', b'23', b'24', b'25', b'26', b'27', b'28', b'29', b'30', b'31']
```

(32 is the max allowed partition count that could be created on azure portal)